docs/release-notes/RELEASE-NOTES-2.0.0-rc.1.md

```diff
@@ -39,9 +41,9 @@ The node user API is documented:
 * WebSocket API endpoints are [specified online][api-doc];
 * The intended usage of the user API (HTTP and WebSocket) is [documented online][api-doc].
 
-[swagger-yaml]: https://github.com/aeternity/aeternity/blob/v1.5.0/config/swagger.yaml
-[swagger-ui]: https://aeternity.github.io/api-docs/?config=https://raw.githubusercontent.com/aeternity/aeternity/v1.5.0/apps/aehttp/priv/swagger.json
-[api-doc]: https://github.com/aeternity/protocol/blob/aeternity-node-v1.5.0/node/api/README.md
+[swagger-yaml]: https://github.com/aeternity/aeternity/blob/v2.0.0-rc.1/config/swagger.yaml
+[swagger-ui]: https://aeternity.github.io/api-docs/?config=https://raw.githubusercontent.com/aeternity/aeternity/v2.0.0-rc.1/apps/aehttp/priv/swagger.json
```

Comment:
```suggestion
[swagger-ui]: https://aeternity.github.io/api-docs/?config=https://raw.githubusercontent.com/aeternity/aeternity/v2.0.0-rc1/apps/aehttp/priv/swagger.json
```

Reply:
I would expect contextual file rename. I plan to merge then review your PR.